common/config/rush/shrinkwrap.yaml

```diff
@@ -28,6 +28,7 @@ dependencies:
   '@rush-temp/test-bundles': 'file:projects/test-bundles.tgz'
   '@rush-temp/test-utilities': 'file:projects/test-utilities.tgz'
   '@rush-temp/theme-samples': 'file:projects/theme-samples.tgz'
+  '@rush-temp/theming-designer': 'file:projects/theming-designer.tgz'
```

Comment:
To resolve the shrinkwrap issue, I'd recommend copying the current shrinkwrap file from master into your branch: `git checkout master -- common/config/rush/shrinkwrap.yaml`. *Then* running `rush generate` (or `npm run update`, which does the same thing). This way the amount of new stuff it has to figure out how to generate will be much smaller.